src/components/TextArea/index.tsx

```diff
@@ -84,14 +83,12 @@ const TextArea: React.FC<IManualControlProps> = props => {
       >
         {!!errorText && !disabled && (
           <View style={styles.errorContainer}>
-            <Icon
-              width={20}
-              height={20}
-              name="error"
-              filled
-              color={theme.error.main}
-            />
-            <ErrorMessage style={styles.errorMessage} error={errorText || ''} />
+            {(errorText?.length ?? 0) > 0 && (
```

Comment:
I think `errorText && (` will work as well

Reply:
`!!errorText && (` also

Reply:
Or maybe even use ternary, because of the 0 result in && that can appear. I switched to it some time ago even though I liked && better.

https://github.com/airbnb/javascript/issues/520#issuecomment-167003046

Reply:
updated - I've removed this one as it's not needed.